frontend/packages/knative-plugin/src/models.ts

```diff
@@ -75,7 +75,7 @@ export const ServiceModel: K8sKind = {
   labelPlural: 'Services',
   plural: 'services',
   id: 'service',
-  abbr: 'S',
+  abbr: 'KSVC',
```

Comment:
Am I missing something? Shouldn't this `S` be `KSVC`?

![image](https://user-images.githubusercontent.com/8126518/70185867-a407f580-16b8-11ea-9f8b-7ace8c689303.png)

Oddly enough, clicking on the resource link at the top of the sidebar after selecting it takes me to Knative Services.

Reply:
I think it might be API provided?

![image](https://user-images.githubusercontent.com/8126518/70188507-e46a7200-16be-11ea-9962-5788300488b0.png)

I passed the model in to see if I could trigger the code you added here to fix it, but it didn't work... what I did see was the _old_ model?

Reply:
Yep, see previous comment. Route and Service are duplicated kinds